### ISSUE TITLE
Feature Request: Account Rating/Feedback System (e.g., Percentage or Stars)

### ISSUE THREAD

**Issue by smarkoco** [retoaccess1/haveno-reto#43]
### Current Problem:
As far as safety insurance for buyers, I only see the account age, and whether an account is "signed", which I understand limits the amount of XMR to be transacted. However, just because an account is old (>60 days) and signed, doesn't mean they couldn't be continually scamming users, right? Each user would have to individually have a bad experience with that account to understand to avoid that user in the future. I'll give an example, based on the most common seller Deposit amount of 15-20%.

An account, Bob, could have a 200 day-old account, and have sold XMR in 1 transaction per day. Imagine that in 20 of those transactions, he never sent the funds after receiving the payment. Each time, the buyer calls on the arbitrators, who recognizes the scam and sends both 15% Deposits to the scammed buyer, but Bob still nets 85% profit.

What's to stop a new buyer from falling into Bob's 21st scam on his next transaction, again netting 85% of the transaction total?

### Possible Solutions:

1. Implement a rating system for each user to submit a review on the other party at the end of the transaction, and include that information within the "Peer info" for new users to see. The rating of an account is simply the average of all previous ratings received. And maybe: text reviews for an account could be viewed in a separate tab within the GUI,

AND/OR

2. Implement a simple blacklist hosted on the Haveno website for arbitrators to add account names to

**Comment by privacyOG** [comment]
Whats stopping the scammer from deleting and reinstalling a new instances haveno/reto. Also the reputation/feedback can be easily manipultaled one example if user has number of PC/Laptops and trades to himself and leaves a number of positive feedback. 

I do understand your concern but the solution will need be different than reputation. 
At moment and how I understood it you must have number of successful trades with no issues requiring arbs to get involved to get a signed account after 60 days. Can that signing of account system get tweeked maybe but no system is perfect.

**Comment by smarkoco** [comment]
I saw a comment on here a few days ago, which appears to have been deleted, but it raised good critiques of my suggestion(s):

First it mentioned the ease of someone recreating new accounts to get around a solution like this. That's a fair point but that's where the account age comes into play. If an account is new, it wouldn't have many ratings yet, and should then be less trusted.

Second, that if a rating system were implemented, people could simply scam this by creating many accounts and transacting with himself over and over to inflate the score. This is also a very fair point I hadn't considered. Maybe only signed accounts or accounts above a certain age threshold could submit reviews? But this could end up making things worse if a scammer gets fully entrenched and has a bunch of old, signed accounts he uses for rating scam purposes

I was just trying to get the conversation started because as a new buyer, I had some bad feelings about going with a Cash by Mail method when all I could see was account age. For Cash by Mail accounts, it said something like "this account type does not require signing", which I didn't understand the reasoning for and it felt less secure.

TL;DR
Really, I would just love to know if there could be any method to proactively root out bad actors and prevent them from using the Cash by Mail method as a method to scam for a long time

**Comment by retoaccess1** [comment]
If Bob tries to scam, he loses 15%. If he tries often with same old account, we can send punishment transaction on the scam offers.

**Comment by smarkoco** [comment]
> If Bob tries to scam, he loses 15%. If he tries often with same old account, we can send punishment transaction on the scam offers.

Thanks, great to hear there is some oversight over time by the arbitrators. It is a complicated problem to solve, nice work 👏
